python/tests/integration_tests/gcp_resources.json

```diff
@@ -0,0 +1,374 @@
+{
+    "bucket": "ops_dev_bucket",
+    "tests": {
+        "list_bucket": {
+            "resources": [
+                {
+                    "path": "list_bucket_test/ex_file_1.txt",
+                    "data": "kw5z3qci2nwrvw9rtgeu7puawfcqak"
+                },
+                {
+                    "path": "list_bucket_test/ex_file_2.txt",
+                    "data": "s3amaa8o56hppijna8p57gijakeiv9"
+                },
+                {
+                    "path": "list_bucket_test/ex_file_3.txt",
+                    "data": "fabbon74i9p7qfbv1tacu9m9a6ligj"
+                }
+            ]
+        },
+        "copy_file": {
+            "resources": [
+                {
+                    "path": "copy_cloud_file_test/original_file.txt",
+                    "data": "p6znd6b4krh6c4g5gg1x4j6cexlw18"
+                }
+            ],
+            "test_data": {
+                "function_input": {
+                    "source_path": "gs://ops_dev_bucket/copy_cloud_file_test/original_file.txt",
+                    "destination_path": "gs://ops_dev_bucket/tmp/file_copied.txt"
+                },
+                "validation": [
+                    {
+                        "Path": {
```

Comment:
Think change just hasn't been committed yet, but this can be `path`?

Reply:
just pushed this! Forgot to save the file after the change.